tests/e2e/framework/kubernetes.go

```diff
@@ -562,13 +568,30 @@ func (k *KubeInfo) Teardown() error {
 			} else {
 				istioYaml = getClusterWideInstallFile()
 			}
-
+		}
+		if *useOperator {
+			// Need an operator unique delete procedure
+			if _, err := util.Shell("kubectl -n %s get IstioControlPlane example-istiocontrolplane -o=json | jq '.metadata.finalizers = null' | kubectl -n %s apply --kubeconfig=%s -f -",
+				 "istio-operator", "istio-operator", k.KubeConfig); err != nil {
+				log.Errorf("Failed to turn off operator finalizer.")
+				return err
+			}
+			if _, err := util.Shell("kubectl delete ns %s --grace-period=0 --force --kubeconfig=%s",
```

Comment:
why grace-period=0?

Reply:
not really sure - these were operations produced by others.

Reply:
this grace period and force don't seem necessary.  I will remove them.  I am told by the people that sorted out this workflow this was their way of forcibly removing resources.

Reply:
grace period and force have been removed.